docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>1.5 looks weird in the ylabel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Fixed in #1290.